### PR TITLE
[VDR] Logging

### PR DESCRIPTION
**Pull request by DenisRybas**
Added logging to VDR using `log` and `env-logger` crates

### CODE REVIEW

File: indy-besu/vdr/src/client/client.rs [hunk 1]
```diff
@@ -31,18 +33,23 @@ impl LedgerClient {
...
-        Ok(LedgerClient {
+
+        let ledger_client = LedgerClient {
             chain_id,
             client: Box::new(client),
             contracts,
-        })
+        };
+
+        info!("Created new LedgerClient");
```
Comment (akarabashov) [inline indy-besu/vdr/src/client/client.rs]: Can we log additional information about the LedgerClient, such as the chain_id and node_address?

File: indy-besu/vdr/src/client/implementation/web3/client.rs [hunk 2]
```diff
@@ -26,10 +27,14 @@ impl Web3Client {
...
         let transport = Http::new(node_address).map_err(|_| VdrError::ClientNodeUnreachable)?;
         let web3 = Web3::new(transport);
-        Ok(Web3Client {
+        let web3_client = Web3Client {
             client: web3,
             signer,
-        })
+        };
+
+        debug!("Created new Web3Client");
```
Comment (akarabashov) [inline indy-besu/vdr/src/client/implementation/web3/client.rs]: Can we log additional information about the created Web3Client?

File: indy-besu/vdr/src/client/implementation/web3/signer.rs [hunk 3]
```diff
@@ -17,7 +18,11 @@ pub struct Web3Signer<'a> {
 
 impl<'a> Web3Signer<'a> {
     pub fn new(account: Address, signer: &'a Box<dyn Signer>) -> Web3Signer<'a> {
-        Web3Signer { account, signer }
+        let signer = Web3Signer { account, signer };
+
+        debug!("Created new signer");
```
Comment (akarabashov) [inline indy-besu/vdr/src/client/implementation/web3/signer.rs]: Can we log additional information about the created signer?

**Comment by Toktar** [comment]
Could we add parameters to logs in methods start?
Action item: check log level for different logs in indy-vdr, indy-sdk. Make the same with our approach.

File: indy-besu/vdr/src/client/implementation/web3/signer.rs (hunk 3 above)
Comment (DenisRybas) [reply indy-besu/vdr/src/client/implementation/web3/signer.rs]: Added info about address

File: indy-besu/vdr/src/client/implementation/web3/client.rs (hunk 2 above)
Comment (DenisRybas) [reply indy-besu/vdr/src/client/implementation/web3/client.rs]: Added node_address

File: indy-besu/vdr/src/client/client.rs (hunk 1 above)
Comment (DenisRybas) [reply indy-besu/vdr/src/client/client.rs]: Added

**Review by Toktar — COMMENTED** [review]
- Did we leave some logs in debug instead of trace on purpose? Is it the same with indy-vdr? 
- Do we want to use ok_or_else instead of ok_or?

**Review by akarabashov — APPROVED** [review]

**Comment by Toktar** [comment]
Could you add trace at the start of function if we log the function output?
Example: LedgerClient::new()

**Review by Toktar — APPROVED** [review]

**Pull request merged**